src/main/java/Attendance.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+
+// Attendance class representing an attendance, with fields date, classSession
+// and student
+public class Attendance {
+
+
+    public Attendance(String date, ArrayList<Student> students) {
+        this.date = date;
+        this.students = students;
+    }
+
+    String date;
+    ArrayList<Student> students;
+    
+    enum AttendanceStatus {
+        PRESENT, ABSENT
+    }
+    static void markAttendance()
```

Comment:
When we create attendance records, we are already marking the attendance, meaning the Event class will be responsible for marking the attendance. Do we need this method ? and should it be static ?